std/conv.d

```diff
@@ -2862,7 +2862,17 @@ unittest
 
     // min and max
     real r = to!real(to!string(real.min_normal));
-    assert(to!string(r) == to!string(real.min_normal));
+    version(NetBSD)
+    {
+        // NetBSD notice
+        // to!string returns 3.3621e-4932L. It is less than real.min_normal and it is subnormal value
+        // Simple C code 
+        //     long double rd = 3.3621e-4932L;
+        //     printf("%Le\n", rd);  
+        // has unexpected result: 1.681050e-4932 
```

Comment:
Actually I suppose there is two issues. First of all this test is naive itself. "to!string" drops digits from long double. So it is actually rounding. Why we think that that after this `assert(to!string(r) == to!string(real.min_normal))` is true? 
The second issue is related libc implementation on NetBSD. You can read answer to my question on SO: [NetBSD long double trouble](http://stackoverflow.com/questions/35090322/netbsd-long-double-trouble) 
So I think it is possible to fix problem by writing dlang implementation for real to string conversation, but it is big task, and it is definitely should not be part of this pull request.